src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/model/Location.kt

```diff
@@ -11,4 +18,57 @@ data class Location(
 
   @Schema(description = "Flag indicates if the location is enabled, true is enabled and false is disabled.", example = "true")
   val enabled: Boolean,
+
+  @Schema(description = "Optional additional attributes about the room and its usage.", example = "true")
```

Comment:
Not sure on this  documentation for the room attributes `example = "true"` ??

Reply:
Sorted. Removed the value (copy error), so room schedule docs apply.